utils/file.go

```diff
@@ -21,6 +21,9 @@ func CopyFile(src, dst string) (err error) {
 	}
 	defer in.Close()
 
+	if err = os.MkdirAll(filepath.Dir(dst), os.ModePerm); err != nil {
+		return
+	}
```

Comment:
This is a no-op if the destination directory already exists, but creates the directory if it doesn't. This is useful when copying files to dynamically created directories, which is something that we do when running the compliance export job.